views/layer/ellipsecolumns.html

```diff
@@ -0,0 +1,47 @@
+<div>
+
+  <div>
+    Applying these mappings will cause the layer to reload and you could lose its current state. <b>This action cannot be undone.</b>
+  </div>
+
+  <div>
+    <div class="form-inline mb-1">
+      <input class="form-check-input" type="radio" ng-value="0" ng-model="ctrl.inputType" id="circle">
+      <label class="form-check-label" for="circle">Circle</label>
```

Comment:
It might be nice to put these radios together at the top of the form. When you change the selection, the Ellipse radio moves because of the changes to the rest of the form. As a user, this is a bit jarring and would feel better if the radios were stationary and only the form below changed.